Fix/Improvement: reworded the wallet methods sentence (#84)
MetaMask/metamask-docs: docs/guide/accessing-accounts.md

# Accessing Accounts

> User accounts are used in a variety of contexts in Ethereum, they serve as neat identifiers. Very important methods to use are the `wallet methods`.

Reviewer: ```suggestion
User accounts are used in a variety of contexts in Ethereum, including as identifiers and for signing transactions. In order to request a signature from the user or have the user approve a transaction, one must be able to access the user's accounts. The `wallet methods` below involve a signature or transaction approval and all require the sending account as a function parameter.
```

> These methods involve a signature or transaction approval.

Reviewer: ```suggestion
```

> These methods involve a signature or transaction approval.

Reviewer: ```suggestion
```

> All of those methods require the sending account as a function parameter:

Reviewer: ```suggestion
```
